src/global/common.rs

```diff
@@ -257,6 +259,104 @@ pub(crate) struct EnvChanges {
     pub changes: HashMap<EnvironmentName, EnvState>,
 }
 
+#[derive(Debug, Default)]
+pub(crate) struct InstallChanges {
+    pub changes: HashMap<PackageName, InstallChange>,
+}
```

Comment:
I would expect this functionality to be part of `StateChanges`

Reply:
as discussed, moved it to StateChanges